app/src/main/res/layout/activity_main.xml

```diff
@@ -16,87 +16,84 @@
         android:text="@string/drag_me"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"/>
+        app:layout_constraintTop_toTopOf="parent" />
 
     <TextView
         android:id="@+id/first"
         style="@style/AppTheme.CircleTextView"
-        android:layout_width="@dimen/circle_size"
-        android:layout_height="@dimen/circle_size"
+        android:layout_width="@dimen/other_circles_size"
+        android:layout_height="@dimen/other_circles_size"
         android:layout_marginTop="@dimen/circle_distance"
         android:backgroundTint="#388e3c"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/drag"/>
+        app:layout_constraintTop_toBottomOf="@+id/drag" />
 
     <TextView
         android:id="@+id/second"
         style="@style/AppTheme.CircleTextView"
-        android:layout_width="@dimen/circle_size"
-        android:layout_height="@dimen/circle_size"
+        android:layout_width="@dimen/other_circles_size"
```

Comment:
As discussed above, since the intention is to provide a dynamic calculation of X and Y, it would be more appropriate to give different `dimen` resource to the `first` and `second` views